src/pages/settings/settings.component.ts

```diff
@@ -26,6 +27,13 @@ export class SettingsComponent {
           this.autoRefresh = '45000';
         }
       });
+      storage.get('departureSort').then(departureSort => {
+        if (departureSort) { 
+          this.departureSort = departureSort;
+        } else {
+          this.departureSort = 'route';
```

Comment:
There are a few places where I set route as the default. So, it's definitely the default. Is this overkill? I don't know — I was mostly copying the auto refresh service.

Reply:
Ehhhhhhh